enterprise/admin/views.py

```diff
@@ -328,7 +362,16 @@ def post(self, request, customer_uuid):
             if course_details:
                 course_mode = manage_learners_form.cleaned_data[ManageLearnersForm.Fields.COURSE_MODE]
                 course_id = course_details['course_id']
-                self._enroll_users(enterprise_customer, linked_learners, course_id, course_mode, request)
+                notification_type = manage_learners_form.cleaned_data.get(ManageLearnersForm.Fields.NOTIFY)
+                notify = notification_type == ManageLearnersForm.NotificationTypes.BY_EMAIL
```

Comment:
Fine by me! Can we treat it as an enum everywhere though? Or maybe even a set :-p (e.g. why should "email" and "SMS", or whatever other options you envision be mutually exclusive). This change is not super important though imho.